src/main/java/cn/nukkit/form/element/custom/ElementDropdown.java

```diff
@@ -17,45 +17,73 @@
 @Accessors(chain = true, fluent = true)
 @AllArgsConstructor
 public class ElementDropdown extends Element implements ElementCustom {
+
     protected String text;
     protected List<String> options;
+    protected String tooltip;
```

Comment:
Make `tooltip` the last element of the constructor to match other elements' constructors.